internal/framework/routes.go

```diff
@@ -0,0 +1,98 @@
+package framework
+
+import (
+	"net/http"
+)
+
+type RouteInfo struct {
+	method  string
+	path    string
+	handler http.HandlerFunc
+}
+
+func NewRouteInfo(method string, path string, handler http.HandlerFunc) *RouteInfo {
+	return &RouteInfo{method: method, path: path, handler: handler}
+}
+
+func (r RouteInfo) Handler() http.HandlerFunc {
+	return r.handler
+}
+
+func (r RouteInfo) Method() string {
+	return r.method
+}
+
+func (r RouteInfo) Path() string {
+	return r.path
+}
+
+type Routes struct {
+	routes []RouteInfo
```

Comment:
I propose to create a system where each module will add its own routes to this object gotten from DIC. And each router will get all routes from this object and create handlers inside itself.

In future requests, it is possible to recreate the routes field as a map. It will add the ability to override or fade routes of any modules you want.